docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -597,3 +597,23 @@ spec:
   resolution: "Enroll device to MDM"
   platforms: macOS
   contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Application is up to date or not present (macOS)
+  query: SELECT 1 WHERE EXISTS (SELECT 1 FROM apps a1 WHERE a1.bundle_identifier = 'com.electron.dockerdesktop' AND a1.bundle_short_version>='4.6.1') OR NOT EXISTS (SELECT 1 FROM apps a2 WHERE a2.bundle_identifier = 'com.electron.dockerdesktop');
+  description: "Checks if the application (Docker Desktop example) is installed and up to date, or not installed. Fails if the application is installed and on a lower version. To check a different application, in the 'Query', replace the bundle_identifier value and the bundle_short_version value. You can copy this query and replace the bundle_identifier and bundle_version values to apply the same type of policy to other applications."
```

```suggestion
  description: "Checks if the application (Docker Desktop example) is installed and up to date, or not installed. Fails if the application is installed and on a lower version. You can copy this query and replace the bundle_identifier and bundle_version values to apply the same type of policy to other applications."
```

I think my suggestion created duplicate instructions here. I like yours better so I'm removing mine!